lib/Http/DownloadResponse.php

```diff
@@ -50,11 +53,11 @@ public function render() {
 					$this->sendNotSatisfiable($size);
```

Comment:
```
					$this->sendNotSatisfiable($size);
```
This method is not defined and causing the phpstan to fail. not sure how to fix this